src/main/java/com/stories/controller/StoriesController.java

```diff
@@ -0,0 +1,29 @@
+package com.stories.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.stories.services.StoryServiceImpl;
+import com.stories.domain.StoryDomain;
+
+@RestController
+@RequestMapping(value = "/stories", produces = "application/json")
+public class StoriesController {
+
+	@Autowired
+	StoryServiceImpl storyService;
+
+	@ResponseStatus(value = HttpStatus.OK)
+	@PutMapping("/StoryUpdate")
```

Comment:
URLs should always be lowercase. This endpoint should not use StoryUpdate, it should expect an ID as path parameter.